app/mailers/participant_validation_mailer.rb

```diff
@@ -105,6 +107,19 @@ def induction_coordinator_email(recipient:, school_name:, start_url:)
     )
   end
 
+  def induction_coordinator_validation_notification_email(recipient:, school_name:, start_url:)
+    template_mail(
+      INDUCTION_COORDINATOR_VALIDATION_NOTIFICATION_TEMPLATE,
+      to: recipient,
+      rails_mailer: mailer_name,
+      rails_mail_template: action_name,
+      personalisation: {
+        school_name: school_name,
+        start_url: start_url,
```

Comment:
Probably going to want a step-by-step url in here?

Reply:
I've asked about this